scripts/start.js

```diff
@@ -48,12 +49,21 @@ let isElectronRunning = false;
  * Prevents multiple re-runs of Electron App
  */
 function runElectronApp() {
-  if (isElectronRunning)
-    return;
+  if (isElectronRunning) return;
 
   isElectronRunning = true;
-
-  exec(`ELECTRON_START_URL=http://localhost:${DEFAULT_PORT} electron .`,
+  // For Windows Support
+  const envPATHS = process.env.PATH.split(';');
+  envPATHS.push(path.join(__dirname, '../node_modules', '.bin'));
+  const newPATHS = envPATHS.join(';');
+  exec(
+    `electron${/^win/.test(process.platform) ? '.cmd' : ''} .`,
```

Comment:
Glad this was a quick fix!

I have a tiny refactor suggested:

```js
const electronCommand = /^win/.test(process.platform) ? 'electron.cmd' : 'electron';

exec(
  `${electronCommand} .`,
  { ... unchanged }
);
```

I think this refactor is somewhat subjective, especially the part about updating the ternary to include `electron` in both sides... but I think the tiny bit of duplication is worth it, since your brain doesn't have to stitch it together, it's done automatically. And giving it a variable makes it clearer what the intent is, + is less easy to miss the fact that it's calling `electron .` (with the `.` for the current directory).